.github/renovate.json5

```
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [ // See what config:recommended does: https://docs.renovatebot.com/presets-config/#configrecommended
+    "config:recommended",
+    // Dependency Dashboards are Issues Renovate creates which list all pending updates. We disable it
+    // because we don't need it, we can see the pending updates in the PRs. This is also noisy.
+    // https://docs.renovatebot.com/configuration-options/#dependencydashboard
+    // Enabled for now to see how it works and if it's useful, we can disable it later if needed.
+    // ":disableDependencyDashboard"
+
+    // Update the digest of the github action workflows to the latest version:
+    "helpers:pinGitHubActionDigests"
+  ],
+
+  // Add pull request labels:
+  "labels": ["dependencies"],
+
+  // Bump even patch versions:
+  "bumpVersion": "patch",
+
+  // Let Renovate decide how to update. See docs: https://docs.renovatebot.com/configuration-options/#rangestrategy
+  "rangeStrategy": "auto",
+
+  // Update the lock files:
+  "lockFileMaintenance": {
+    "enabled": true,
+    "recreateWhen": "always",
+    "rebaseStalePrs": true,
+    "branchTopic": "lock-file-maintenance",
+    "commitMessageAction": "Lock file maintenance",
+    "commitMessageTopic": null,
+    "commitMessageExtra": null,
+    "schedule": ["* * * * 1,4"],  // Run sometime on Monday and Thursday
+    "groupName": "Bump versions in lock files",
+    "prBodyDefinitions": {
+      "Change": "All locks refreshed"
+    }
+  },
```

```suggestion
  "lockFileMaintenance": {
    "enabled": true,
    "schedule": ["* * * * 1,4"],  // Run sometime on Monday and Thursday
  },
```
The removals are either consistent with the default behavior or differ from it only in wording.